ext/active_record/associations/association.rb

```diff
@@ -2,7 +2,7 @@ module ActiveRecord
   module Associations
     class Association #:nodoc:
 
-      def skip_statement_cache?
+      def skip_statement_cache?(scope)
```

Comment:
Shouldn't this statement reflect the 5.2 version? 

https://github.com/rails/rails/blob/5-2-stable/activerecord/lib/active_record/associations/association.rb#L282-L287